stockroom/external/importer/torchvision_importers.py

```diff
@@ -26,7 +26,7 @@ def dtypes(self):
     @staticmethod
     def _process_data(img, lbl):
         # TODO: memory copy
-        img = np.ascontiguousarray(np.transpose(np.array(img), (2, 0, 1)))
+        img = np.ascontiguousarray(np.array(img))
```

Comment:
you don't need `np.ascontiguousarray` here. The array is already contiguous

Reply:
yep, changed